debian/changelog

```diff
@@ -0,0 +1,5 @@
+tvb-gdist (1.5.6-1) unstable; urgency=medium
+
+  * Initial release. Closes: #896927
```

Comment:
What does #896927 mean ?
I suggest leaving only "initial debian release"

Reply:
Is a bug number(intend to package ITP) here it is https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=896927